src/index.html

```diff
@@ -4,9 +4,415 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a class="header__link" href="/">
+          <picture>
+            <source srcset="/images/logo.png" media="(orientation: portrait)" >
+            <img src="/images/logo.png" alt="Logo" >
+          </picture>
+        </a>
+      </div>
+      <div class="header__block">
+        <nav class="header__navigation">
+          <ul class="header__menu">
+            <li class="header__navitem">
+              <a class="header__navlink is-active">Apple</a>
+            </li>
+
+            <li class="header__navitem">
+              <a class="header__navlink">Samsung</a>
+            </li>
+
+            <li class="header__navitem">
+              <a class="header__navlink">Smartphones</a>
+            </li>
+
+            <li class="header__navitem">
+              <a class="header__navlink" data-qa="nav-hover">Laptops & Computers</a>
+            </li>
+
+            <li class="header__navitem">
+              <a class="header__navlink">Gadgets</a>
+            </li>
+
+            <li class="header__navitem">
+              <a class="header__navlink">Tablets</a>
+            </li>
+
+            <li class="header__navitem">
+              <a class="header__navlink">Photo</a>
+            </li>
+
+            <li class="header__navitem">
+              <a class="header__navlink">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="card-grid">
+        <div class="card" data-qa="card">
+          <img src="./images/imac.jpeg" alt="Product image"
+          class="card__img">
```

Comment:
Pay attention on code style. Fix all simila cases
In case tag has more than 2 attributes - each atribute should be from new line, should have 2 spaces, closed bracket should be on the same line as open.
```suggestion
          <img
            src="./images/imac.jpeg"
            alt="Product image"
            class="card__img"
          >
```